src/Symfony/Component/HttpClient/HttpClient.php

```diff
@@ -28,12 +30,20 @@ final class HttpClient
      *
      * @see HttpClientInterface::OPTIONS_DEFAULTS for available options
      */
-    public static function create(array $defaultOptions = [], int $maxHostConnections = 6): HttpClientInterface
+    public static function create(array $defaultOptions = [], LoggerInterface $logger = null, int $maxHostConnections = 6): HttpClientInterface
     {
+        if ($logger === null) {
+            $logger = new NullLogger();
+        }
+
         if (\extension_loaded('curl')) {
-            return new CurlHttpClient($defaultOptions, $maxHostConnections);
+            $logger->info('Curl extension is enabled. Creating client.', ['client' => CurlHttpClient::class]);
```

Comment:
this should be at the `debug` level IMO rather than at `info`, to be able to exclude it more easily (that's not a very useful log message most of the time)